### PR TITLE
[PLAYER-5441] Fix test runner, add scripts to generate coverage report

### PR DESCRIPTION
**Pull request by loginov-rocks**
As mentioned in the subject. Also created few simple tests to show how it works. The aim of this task was not to cover something with tests, just to properly configure and test Jest integration.

This is how the coverage report looks like:
Screen Shot 2019-04-02 at 1 00 51 PM

### CODE REVIEW

File: sdk/react/src/markers/IconMarker/IconMarker.test.js [hunk 1]
```diff
@@ -1,32 +1,67 @@
...
+    const onSeekMock = jest.fn();
+    const testRenderer = TestRenderer.create(
+      <IconMarker
+        iconUrl="http://example.com/icon.png"
+        imageUrl="http://example.com/image.png"
+        onSeek={onSeekMock}
+      />,
+    );
+
+    testRenderer.root.findByType(TouchableWithoutFeedback).props.onPress();
```
Comment (ooyala-amohnatkin) [inline sdk/react/src/markers/IconMarker/IconMarker.test.js]: isn't it better simulate press than call component method directly?

Comment (loginov-rocks) [reply sdk/react/src/markers/IconMarker/IconMarker.test.js]: How can I do it?

Comment (ooyala-amohnatkin) [reply sdk/react/src/markers/IconMarker/IconMarker.test.js]: try `testRenderer.root.findByType(TouchableWithoutFeedback).simulate('press');`

File: sdk/react/src/markers/IconMarker/IconMarker.test.js [hunk 2]
```diff
@@ -1,32 +1,67 @@
...
+import React from 'react';
+import { TouchableWithoutFeedback } from 'react-native';
+import TestRenderer from 'react-test-renderer';
 
 import IconMarker from './IconMarker';
 
 describe('IconMarker', () => {
   it('renders matching snapshot', () => {
-    const tree = renderer.create(
+    const testRenderer = TestRenderer.create(
```
Comment (ooyala-amohnatkin) [inline sdk/react/src/markers/IconMarker/IconMarker.test.js]: `testRenderer` looks confusing as it's not a renderer but its instance.
As for me, `testInstance`, `element` or `wrapper` are more convenient

Comment (loginov-rocks) [reply sdk/react/src/markers/IconMarker/IconMarker.test.js]: Agree, I spent sometime on naming and just took what Facebook uses. Will use `wrapper` as common name in Enzyme.

Comment (loginov-rocks) [reply sdk/react/src/markers/IconMarker/IconMarker.test.js]: Fixed

File: sdk/react/src/markers/IconMarker/IconMarker.test.js (hunk 1 above)
Comment (loginov-rocks) [reply sdk/react/src/markers/IconMarker/IconMarker.test.js]: `TypeError: wrapper.root.findByType(...).simulate is not a function` it seems to be deprecated as no mentions are present at https://reactjs.org/docs/test-renderer.html Some tutorials I found uses this approach with `props.onClick()`

Comment (loginov-rocks) [reply sdk/react/src/markers/IconMarker/IconMarker.test.js]: Moved to Enzyme, @ooyala-amohnatkin please review

**Review by ooyala-amohnatkin — APPROVED** [review]

**Review by SergeyBicarte — APPROVED** [review]

**Pull request merged**